monai/apps/mmars/mmars.py

```diff
@@ -126,27 +138,65 @@ def load_from_mmar(item, mmar_dir=None, progress: bool = True, map_location=None
     # loading with `torch.load`
     model_dict = torch.load(model_file, map_location=map_location)
     if weights_only:
-        return model_dict["model"]
-
-    # TODO: search for the module based on model name?
-    if not model_dict.get("train_conf", ""):
-        raise ValueError("The MMAR configuration does not have a 'train_conf' section.")
-    model_config = model_dict["train_conf"]["train"]["model"]
-    if model_config.get("name", ""):  # model config section is a "name"
+        return model_dict.get("model", model_dict)  # model_dict["model"] or model_dict directly
+
+    # 1. search `model_dict['train_config]` for model config spec.
+    model_config = _get_val(dict(model_dict).get("train_conf", {}), key=model_key, default={})
+    if not model_config:
+        # 2. search json CONFIG_FILE for model config spec.
+        json_path = os.path.join(model_dir, item.get(Keys.CONFIG_FILE, "config_train.json"))
+        with open(json_path) as f:
+            conf_dict = json.load(f)
+        conf_dict = dict(conf_dict)
+        model_config = _get_val(conf_dict, key=model_key, default={})
+    if not model_config:
+        # 3. search `model_dict` for model config spec.
+        model_config = _get_val(dict(model_dict), key=model_key, default={})
```

Comment:
May I know what this case is for? I think Clara `model.pt` doesn't have model config in it directly.

Thanks.

Reply:
this is the last guess for the model config location, in case that `model_dict` doesn't have a `train_conf` section, but have something similar, for example a `training_conf` section.

Reply:
OK, I see.
Clara doesn't support this case so far. And if it is, how can I specify the `training_conf` key in this case?

Thanks.

Reply:
the `_get_val` is a search in the dictionary for `model_key`, the idea is to not assuming the `train_conf` or `training_conf`

Reply:
OK, that makes sense!
Thanks.